pinot-plugins/pinot-segment-writer/pinot-segment-writer-file-based/src/test/java/org/apache/pinot/plugin/segmentwriter/filebased/FileBasedSegmentWriterTest.java

```diff
@@ -119,15 +119,16 @@ public void testBatchConfigs()
       // expected
     }
 
-    ingestionConfig.setBatchIngestionConfig(new BatchIngestionConfig(null, "APPEND", "HOURLY"));
+    ingestionConfig.setBatchIngestionConfig(new BatchIngestionConfig(null, "APPEND", "HOURLY", false, false));
```

Comment:
To avoid these multiple touch points in preexisting code that does not use consistent data push we can use the Java implementation default arguments: see [Java def args ](https://stackoverflow.com/questions/997482/does-java-support-default-parameter-values)